docs/content/installation.md

```diff
@@ -228,17 +228,18 @@ Wakame-vdc's default directory for keeping images is `/var/lib/wakame-vdc/images
 sudo mkdir -p /var/lib/wakame-vdc/images
 ```
 
-Now download the image in that directory.
+Now download the image in that directory. For a passwordless login image, replace ```passwd-login-enabled``` by ```passwd-login-disabled``` in the link.
```

Comment:
I think it's not entirely clear what passwordless login means.

It might be better to have a quick paragraph explaining what exactly we mean by passwordless login. Just a quick line or two explaining that we offer two images that are identical other than that one allows SSH login using password while another only allows public/private keypair login.